application/src/main/java/module-info.java

```diff
@@ -47,6 +47,7 @@
     requires nsmenufx;
     requires fr.brouillard.oss.cssfx;
     requires org.carlfx.cognitive;
+    requires com.pixelduke.fxcomponents;
```

Comment:
Does the main project need `fxcomponents` or is this for the sampler? If it is for the Sampler we should move it into the test module.

Reply:
Good catch. That was a mistake. Done!

Reply:
> Does the main project need `fxcomponents` or is this for the sampler? If it is only for the Sampler we should remove from here and move it into the test module.

Good catch. That was a mistake. Done!